### DISCUSSION TITLE
Integrate Piper with Balabolka

### DISCUSSION THREAD

**Discussion by MatheusGiusti** [rhasspy/piper#468] [Ideas]
Hello, 
I suggest making Piper usable in Balabolka, via SAPI 5, so that real-time reading of the text is possible.
Thanks.

**Comment by rmcpantoja** [comment]
Hi @MatheusGiusti, 
Just these days I'm making an alternative, something like a balabolka for neural TTS systems. The same has been done with RVC, and it would be very nice to do it in TTS. It's fully realtime. Atm you can test it from the source code, I've tried to compile it, but I'm getting many errors in the process due to the compilation engine.

https://github.com/rmcpantoja/tTS-remix
